examples/reference-entity-model/steps/mapping/mapCustomersJSON.step.json

```diff
@@ -8,7 +8,9 @@
   "selectedSource": "collection",
   "sourceQuery": "cts.collectionQuery(['loadCustomersJSON'])",
   "collections": [
-    "default-mapping"
+    "default-mapping",
```

Comment:
Please remove "default-mapping" too, it's not needed for anything. We don't want to include anything that's not needed, as it'll raise the question of "Why is this here?"